frontend/src/apps/qotd/containers/Navigation.js

```diff
@@ -1,22 +1,15 @@
-import React from 'react'
 import { connect } from 'react-redux'
 import { createSelector } from 'reselect'
 
 import { selectOrchestration } from 'core/app/reducers'
 import { selectApps } from 'core/app/reducers/orchestrationReducer'
 
-class Navigation extends React.Component {
-  render () {
-    return this.props.render()
-  }
+function Navigation(props) {
+  return props.render()
 }
 
-const selector = createSelector(
-  selectOrchestration,
-  selectApps,
-  apps => ({
-    apps
-  })
-)
+const selector = createSelector(selectOrchestration, selectApps, (apps) => ({
+  apps,
+}))
```

Comment:
You should not alter code that is outside the scope of the commit. I suspect the change is a result of autoformatting in your editor.

This will no longer be an issue once we implement a code formatter (i.e. Prettier).